scripts/actions/check-for-outdated-translations.js

```diff
@@ -0,0 +1,118 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const frontmatter = require('@github-docs/frontmatter');
+const parseLinkHeader = require('parse-link-header');
+
+const checkArgs = require('./utils/check-args');
+const { prop } = require('../utils/functional');
+const { ADDITIONAL_LOCALES } = require('../utils/constants');
+
+const doI18nFilesExist = (fileName, locales) => {
+  const i18nPrefix = path.join(process.cwd(), 'src/i18n/content');
+  const baseFileName = fileName.replace('src/content/', '');
+
+  return locales
+    .map((locale) => {
+      const filePath = path.join(i18nPrefix, locale, baseFileName);
+      const fileExists = fs.existsSync(filePath);
+      return fileExists ? filePath : null;
+    })
+    .filter(Boolean);
```

Comment:
This might be simplified by using [`some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some):

```suggestion
  return locales
    .some((locale) => {
      const filePath = path.join(i18nPrefix, locale, baseFileName);
      return fs.existsSync(filePath);
    })
```

Reply:
The file names are being used later on in the script to provide output for the user. It's probably worth changing the function name to `findI18nFiles` instead of `doI18nFilesExist`